src/util/mod.rs

```diff
@@ -5,6 +5,8 @@ use std::collections::HashMap;
 pub(crate) fn build_map<'key, 'value, const N: usize>(
     array: [(&'key str, Option<&'value str>); N],
 ) -> HashMap<&'key str, &'value str> {
+    // Use a manual for loop instead of iterators so we can call `with_capacity` and avoid
```

Comment:
We're currently limiting comments to 80 characters. If you don't like that rule we could definitely change it, though, as I understand it might be annoying to new contributors, and it's hard to enforce automatically.